docs/contributing.rst

```diff
@@ -74,35 +75,37 @@ Any extra texts (print statements and so on) should be removed.
 Tests coverage
 --------------
 
-We are strongly keeping our test coverage, please don't make it worse.
+We are strongly trying to have a good test coverage, 
+please don't make it worst.
```

Comment:
This would read more naturally as "We are trying hard to keep test coverage high; please don't make it worse."